scripts/bb-test-cleanup.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+# Check for a local cached configuration.
+if test -f /etc/buildslave; then
+    . /etc/buildslave
+fi
+
+set -x
+
+if [ -z "$CODECOV_TOKEN" -o \
+     -z "$BUILDER_NAME" -o \
+     -z "$BUILD_NUMBER" -o \
+     -z "$ZFS_REVISION" -o \
+     -z "$BASE_BRANCH" ]; then
+    echo "Missing a required environment variable."
+    exit 1
+fi
+
+GCOV_KERNEL="/sys/kernel/debug/gcov"
+ZFS_BUILD=$(readlink -f ../zfs)
+
+if $(sudo -E test ! -e "$GCOV_KERNEL"); then
+    echo "Kernel Gcov disabled.  Skipping test cleanup."
+    exit 3
+fi
+
+function upload_codecov_reports
+{
+    pushd "${ZFS_BUILD}" >/dev/null
+    curl -s https://codecov.io/bash | bash -s - \
+        -c -Z -X gcov -X py -X xcode \
+        -n "$BUILDER_NAME" \
+        -b "$BUILD_NUMBER" \
+        -C "$ZFS_REVISION" \
+        -B "$BASE_BRANCH" \
+        -P "$PR_NUMBER" \
+        -F "$1"
```

Comment:
this won't hurt anything, but we're not using the "$1" parameter in the current iteration of this script.. I was using it when testing an older version.. thus, we can remove this to avoid any confusion.. leaving it won't hurt, it's just dead code.